core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -56,7 +60,7 @@ <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         </li>
         {% if not can_create_collections %}
           <!-- A layout fix when the 'My Collections' tab is absent -->
-          <li ng-if="collectionsList.length == 0"></li>
+          <li ng-if="collectionsList.length === 0 && subscribersList.length === 0"></li>
```

Comment:
This is needed to maintain the layout. For example if the user has no collections but has subscribers, this condition maintains the layout of the tab. Without `&& subscribersList.length === 0` the layout looks distorted:
![image](https://cloud.githubusercontent.com/assets/13691317/22859472/8f0f1698-f0d4-11e6-8b31-62d1118e7c5a.png)
With the condition, the layout looks good:
![image](https://cloud.githubusercontent.com/assets/13691317/22859474/a52d1754-f0d4-11e6-8674-571cc28a7ae6.png)